### PR TITLE
Feature549/UI biometrics implemented

### PR DESCRIPTION
**Pull request by StefanoPitto**

### CODE REVIEW

File: src/containers/SettingsContainer.tsx [hunk 1]
```diff
@@ -28,13 +29,23 @@ export default function SettingsContainer({ navigation }: Props) {
...
+                            title={'Fingerprint'}
+                            icon={'fingerprint'}
+                        />
+                    ) : (
+                        <TNavigationButton
+                            onPress={() => {
+                                navigation.navigate('FaceIdSettings');
+                            }}
+                            title={'FaceID'}
+                            icon={<FaceIdIcon style={{ maxWidth: 24, marginHorizontal: 14 }} />}
```
Comment (sadiabbasi) [inline src/containers/SettingsContainer.tsx:46]: Please avoid inline styling..

File: src/components/TNavigationButton.tsx [hunk 2]
```diff
@@ -19,11 +20,12 @@ export default function TNavigationButton(props: NavigationButtonProps) {
             style={[styles.button, commonStyles.marginBottom]}
             disabled={props.disabled}
         >
-            {props.icon && (
+            {props.icon && typeof props.icon === 'string' && (
```
Comment (sadiabbasi) [inline src/components/TNavigationButton.tsx:23]: Why you added typeof here?

**Review by sadiabbasi — CHANGES_REQUESTED** [review]

File: src/containers/SettingsContainer.tsx (hunk 1 above)
Comment (StefanoPitto) [reply src/containers/SettingsContainer.tsx]: removed inline styling

File: src/components/TNavigationButton.tsx (hunk 2 above)
Comment (StefanoPitto) [reply src/components/TNavigationButton.tsx:23]: to check if i receive a string or a Component as an icon

**Review by sadiabbasi — APPROVED** [review]

**Pull request merged**